pkgs/applications/graphics/oculante/default.nix

```diff
@@ -69,10 +67,8 @@ rustPlatform.buildRustPackage rec {
   postInstall = ''
     install -Dm444 $src/res/oculante.png -t $out/share/icons/hicolor/128x128/apps/
     install -Dm444 $src/res/oculante.desktop -t $out/share/applications
-  '';
-
-  postFixup = lib.optionalString stdenv.isLinux ''
-    patchelf $out/bin/oculante --add-rpath ${lib.makeLibraryPath [ libxkbcommon libX11 ]}
+    wrapProgram $out/bin/oculante \
```

Comment:
What's the reason for this change?

Reply:
I think this: https://github.com/NixOS/nixpkgs/issues/276729

Reply:
I doubt it ever worked honestly, if it did then by luck of having some env vars set up properly via desktop environment. Even on 2e59fd5ffc1a15f5a71d1593a8969f9608edca0b I get libgl errors on startup